tests/sss/test_federated_metrics.py

```diff
@@ -8,6 +8,94 @@
 from managedtenants.core.addon_manager import AddonManager
 from managedtenants.core.addons_loader.sss import Sss
 
+@given(data=hypothesis_strategies.data())
+def test_metrics_federation_v1(data):
```

Comment:
This was existing functionality right? You just added tests for them right?

Reply:
These tests were added to ensure that the selectorsyncsets are rendered properly. Just added them for the sake of better coverage in favor of backwards compatibility. Although it's pretty much not possible for us to move back to SelectorSyncSet, no harm in having some extra tests ;)